frontend/pages/faq.tsx

```diff
@@ -14,53 +15,37 @@ import HeaderImage from "../src/components/staticpages/HeaderImage";
 
 const useStyles = makeStyles((theme) => {
   return {
-    headerImageContainer: {
-      marginBottom: theme.spacing(3),
-      [theme.breakpoints.down("lg")]: {
-        marginBottom: theme.spacing(2),
-      },
-    },
     headerTextContainer: {
       position: "absolute",
-      top: "50%",
-      left: "50%",
-      transform: "translate(-50%, -50%)",
-      background: "rgba(255, 255, 255, 0.6)",
-      width: "100%",
-      height: "100%",
-      [theme.breakpoints.down("lg")]: {
-        background: "rgba(255, 255, 255, 0.8)",
-      },
+      top: "45%",
+      left: "10%",
+      maxWidth: "400px",
+      minWidth: "250px",
+      borderRadius: "5px",
+      backgroundColor: "#207178",
```

Comment:
position absolute can easily break on different screen sizes. I would rather prefer a different solution, which inserts into the normal layout and then uses `transform` to push it up:

```suggestion
            display: "flex",
      alignItems: "left",
      transform: "translateY(-50%)",
 ```
In this case, using this class and its related div as a wrapper for the row segment makes more sense. Therefore, I would move the `backgroundColor` and `borderRadius` to the div below.